lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1104,7 +1141,7 @@ sub process_and_run_upgrade_script {
         template => $template,
         no_auto_output => 1,
         format_options => {extension => 'sql'},
-        output_file => 'upgrade',
+        output_file => 'upgrade.sql',
```

Comment:
I thought so too and would welcome it. But it should be the object of another PR for it will bring unrelated files into this one and remove the focus.